task-3/services/logging.ts

```diff
@@ -0,0 +1,22 @@
+class LoggingService {
+	scope: string;
+
+	constructor(scope: string) {
+		this.scope = scope;
+	}
+
+	routeLogger = (req: any, res: any, next: any) => {
```

Comment:
While this is already much better in terms of the usability, it's still not that common for loggers to be aware of the implementation details of where they're used.
Approving the PR, but do pay attention to the concept of "Log Levels" and do some further reading on it.
https://blog.appsignal.com/2021/09/01/best-practices-for-logging-in-nodejs.html